src/styles/create-form/create-form.css

```
@@ -0,0 +1,20 @@
+.create-form {
+  background: var(--quizcard);
+  border-color: var(--color-white);
+  color: var(--color-blue);
+  display: flexbox;
+  grid-gap: 12px;
```

It's either `display: flex;` or `display: grid;`. Also `grid-gap` only goes with `display-grid`. These styles don't do much though if you give it to the section around your form. If you give this class to the `form` element, it will be as you thought it would.
```suggestion
  display: grid;
  grid-gap: 12px;
```

---

Changed it. thanks.